recipe/meta.yaml

```diff
@@ -1,31 +1,26 @@
 # Some useful notes can be found at https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT
 
-{% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
-
-# The version should be in the form X.Y, where X denotes major infrastructural change to how blas is managed,
-# and Y denoting blas priority. Currently Y=0 not default, Y=1 default blas.
-{% set Y = 1 if variant == 'openblas' else 0 %}
 package:
   name: blas
-  version: 1.{{ Y }}
+  version: 2.0
 
 build:
   # We mustn't use the build number in the blas package - because we are defining the string,
   # manually we would not be producing unique filenames.
   number: 0
-  string: {{ variant }}
+  string: {{ blas }}
+  run_exports:
+    - blas 2.0.*={{ blas }}
```

Comment:
Should this be `2.0.*` or `2.*`?

Reply:
Good catch.  I think this actually shouldn't be here at all.  Other things (such as an openblas-devel package) should have a run_exports that adds blas-2.something-openblas.  The blas metapackage itself shouldn't generally be directly added to anything.

This is harmless, though, so I'm OK with leaving it here in this PR and cleaning it up soon.  Up to you, though.

Reply:
What I was not sure about was, does `blas 2.0.*=openblas` match `blas 2.0=openblas` (package version as in the PR is 2.0`

Reply:
I removed the run_exports here, as suggested